src/Datadog.Trace.ClrProfiler.Managed/Integrations/ElasticsearchNet6Integration.cs

```diff
@@ -28,9 +30,9 @@ public static class ElasticsearchNet6Integration
         /// <param name="moduleVersionPtr">A pointer to the module version GUID.</param>
         /// <returns>The original result</returns>
         [InterceptMethod(
-            CallerAssembly = "Elasticsearch.Net",
-            TargetAssembly = "Elasticsearch.Net",
-            TargetType = "Elasticsearch.Net.IRequestPipeline",
+            CallerAssembly = ElasticsearchAssemblyName,
+            TargetAssembly = ElasticsearchAssemblyName,
+            TargetType = RequestPipelineInterfaceName,
```

Comment:
Interfaces are types (as are classes and structs). Consider calling this `RequestPipelineTypeName` for consistency with other target types.

Reply:
I wanted it to be clear that it was an interface, and interfaces are somewhat special in the CLR. Happy to change to `RequestPipelineInterfaceTypeName` if that works for you.